runtime/src/main/scala/akka/grpc/internal/ProtoMarshaller.scala

```diff
@@ -1,38 +0,0 @@
-/*
- * Copyright (C) 2018-2020 Lightbend Inc. <https://www.lightbend.com>
- */
-
-package akka.grpc.internal
-
-import java.io.{ ByteArrayInputStream, ByteArrayOutputStream, InputStream }
-
-import io.grpc.KnownLength
-import akka.annotation.InternalStableApi
-import akka.grpc.ProtobufSerializer
-
-/**
- * INTERNAL API
- */
-@InternalStableApi
```

Comment:
We can't remove this API since it's used from generated clients, and we want people to be able to upgrade the runtime library even when downstream libraries used a previous version to generate their clients. https://doc.akka.io/docs/akka-grpc/current/binary-compatibility.html

Reply:
I'm not removing it, I'm only reimplementing it. First I moved this code into another file, and then I extracted the common code. MiMa is not complaining. :-)

Reply:
correction: mima is complaining but the issue is unrelated to this discussion 😅

Reply:
> I moved this code into another file

oops, sorry, missed that